src/sentry/api/endpoints/group_ai_summary.py

```diff
@@ -109,9 +113,10 @@ def post(self, request: Request, group: Group) -> Response:
         if not features.has("organizations:ai-summary", group.organization, actor=request.user):
             return Response({"detail": "Feature flag not enabled"}, status=400)
 
-        if group.data.get("issue_summary"):
+        cache_key = "ai-group-summary:" + str(group.id)
+        if cache.has_key(cache_key):
```

Comment:
You can just use `cache.get` once, like: 
https://github.com/getsentry/sentry/blob/605c24c35845a004926623d9c3c734c1046f62e2/src/sentry/http.py#L165-L166